pymc3/distributions/distribution.py

```diff
@@ -194,7 +194,10 @@ def _str_repr(self, name=None, dist=None, formatting="plain"):
             )
 
     def __str__(self, **kwargs):
-        return self._str_repr(formatting="plain", **kwargs)
+        try:
+            return self._str_repr(formatting="plain", **kwargs)
+        except:
+            return super().__str__()
```

Comment:
Hey @Spaak - is there any example that hits this, or is it here just to be safe?

Reply:
Before the more specific fixes in this PR, `KroneckerNormal` and `MatrixNormal` would have hit this. So for now I think this is just in there to be safe. (We don't want code failing altogether simply because a particular `Distribution` subclass does not play well with the "new" `str` code, also because people might want to use custom subclasses I guess and not have to worry about this.)